### ISSUE TITLE
Bug:  Array.count format to Array.isEmpty

### ISSUE THREAD

**Issue by yanpanpan** [nicklockwood/SwiftFormat#439]
origin code:
``` swift
func test() {
    let a: [Int] = []
    let b: [Int] = []
    if a.count - b.count > 0 {
        print("a > b ")
    }
}
```
will format to:

```swift
func test() {
    let a: [Int] = []
    let b: [Int] = []
    if a.count - !b.isEmpty {
        print("a > b ")
    }
}
```
`a.count - b.count > 0` become  ` a.count - !b.isEmpty `, is not right.   
It should become `a.count > b.count` or leave as it was.

**Comment by nicklockwood** [comment]
Thanks for reporting. I'll get this fixed asap. In the meantime, you can put the following comment directive on the previous line to work around the problem:

```swift
// swiftformat:disable:next isEmpty
```

**Comment by nicklockwood** [comment]
Fixed in version 0.40.9.
